### ISSUE TITLE
Tentang Kami page

### ISSUE THREAD

**Issue by zainfathoni** [kawalcovid19/wargabantuwarga.com#326]
## UI Design

On Figma: https://www.figma.com/file/XNNtIoFEdFqaXOee83n0oN/WBW?node-id=458%3A2632

## Implementation Model

Static Markdown file, editable from Netlify CMS, rendered as static page on Next.js.

## Tasks

- [ ] Initial content for Tentang Kami page with Netlify CMS
- [ ] Create design for Tentang Kami page
- [ ] Create markdown file + content widgets for Tentang Kami page on Netlify CMS
- [ ] Create page on frontend

**Comment by resir014** [comment]
Tasks list updated. Waiting on design from UX team cc: @baraeb92

**Comment by bungmomomm** [comment]
- Create design for Tentang Kami page

Siap ndan 86 @zainfathoni @baraeb92

**Comment by resir014** [comment]
To make this implamentation easier (and to fully complete the homepage transition from gdocs/netlify CMS content), I'm going to implement an initial text page on Netlify CMS with content based on parts of the homepage:

![image](https://user-images.githubusercontent.com/5663877/127841073-10c350a4-9293-42f4-9299-c7956f8e5e25.png)

![image](https://user-images.githubusercontent.com/5663877/127841145-27f5b7b0-f4df-4c54-8b94-33a2f5e984c2.png)

cc: @zainfathoni

**Comment by mazipan** [comment]
![Screen Shot 2021-08-10 at 10 39 42](https://user-images.githubusercontent.com/7221389/128804782-8a266ae3-b347-4125-946e-31b321b0228c.png)

Which URL should be used from this button @zainfathoni?

**Comment by baraeb92** [comment]
We can use this link as we do in homepage @mazipan https://kcov.id/wbw-discuss

**Comment by mazipan** [comment]
Help to completing all the Logos on the Figma @baraeb92

**Comment by baraeb92** [comment]
What are the logos that are still missing ya mas @zainfathoni ? do we miss anyone on the WBW?

**Comment by mazipan** [comment]
Opening new issue on #627 @baraeb92
